tests/testthat/test-Data-methods.R

```diff
@@ -236,3 +236,18 @@ test_that("Update of DataDA works as expected", {
   expect_valid(result, "DataDA")
   expect_identical(result, object)
 })
+
```

Comment:
what about the case when there is no real update because empty vectors? can you also add a test for that?